.changeset/red-months-cough.md

```diff
@@ -0,0 +1,5 @@
+---
+"@navikt/ds-react": patch
+---
+
+Avoid warning about element.ref in React 19
```

Comment:
Since Slot is an internal component, I don't think it's relevant to mention it. We could mention every component that uses it, but then I think the text would become too long 😅